transform/transformers/PDFTransformer.py

```diff
@@ -28,9 +30,17 @@
 styles = getSampleStyleSheet()
 
 # Basic text style
-styleN = styles["BodyText"]
+styleN = copy(styles["BodyText"])
```

Comment:
Are you aware that this is only returning a shallow copy of `styles["BodyText"]` the difference being:


    - A shallow copy constructs a new compound object and then (to the extent possible) inserts references into it to the objects found in the original.
    - A deep copy constructs a new compound object and then, recursively, inserts copies into it of the objects found in the original.

You might want to consider which to use here (though that's not to say this is the wrong choice!)